cds/config.py

```diff
@@ -1672,3 +1672,7 @@ def _parse_env_bool(var_name, default=None):
 
 # Sets the location to share the video files among the different tasks
 CDS_FILES_TMP_FOLDER = "/tmp/videos"
+
+
+# TODO: needs latest files-rest enabling range requests
+FILES_REST_ALLOW_RANGE_REQUESTS = True
```

Comment:
This is needed for local development for now. The easiest solution is to backport that commit in the v2.x releases...